src/adapters/google-maps.adapter.ts

```diff
@@ -59,12 +59,12 @@ export class TerraDrawGoogleMapsAdapter extends TerraDrawBaseAdapter {
 
 		const ne = bounds.getNorthEast();
 		const sw = bounds.getSouthWest();
-		const latLngBounds = new google.maps.LatLngBounds(sw, ne);
+		const latLngBounds = new this._lib.LatLngBounds(sw, ne);
```

Comment:
Thanks for noticing these - this was not intentional and should have been `this._lib` from the start!